src/main/java/io/vertx/core/http/impl/pool/Pool.java

```diff
@@ -81,6 +81,16 @@
     long weight;         // The weight that participates in the pool weight
 
   }
+
+  /**
+   * Defines the policy in which connections will be reused in the pool
+   *
+   */
+  public enum ConnectionRecyclePolicy {
```

Comment:
move it to the `io.vertx.core.http` package instead and rename it just `RecyclePolicy`